docs/collector.update.md

```diff
@@ -22,23 +26,23 @@ Define the interval of scraping Windows Update information
 
 ## Metrics
 
-| Name                           | Description                                   | Type  | Labels                        |
-|--------------------------------|-----------------------------------------------|-------|-------------------------------|
-| `windows_updates_pending_info` | Expose information single pending update item | gauge | `category`,`severity`,`title` |
-| `windows_updates_scrape_query_duration_seconds` | Duration of the last scrape query to the Windows Update API | gauge |  |
-| `windows_updates_scrape_timestamp_seconds` | Timestamp of the last scrape | gauge |  |
+| Name                           | Description                                                 | Type  | Labels                        |
+|--------------------------------|-------------------------------------------------------------|-------|-------------------------------|
+| `windows_update_pending_info` | Expose information for a single pending update item         | gauge | `category`,`severity`,`title` |
+| `windows_update_scrape_query_duration_seconds` | Duration of the last scrape query to the Windows Update API | gauge |  |
+| `windows_update_scrape_timestamp_seconds` | Timestamp of the last scrape                                | gauge |  |
```

Comment:
consider updating the description of the metrics, if you like

https://github.com/prometheus-community/windows_exporter/blob/cb1769cacc2b054a8999bbb19b7693f9dd4f9cc5/internal/collector/update/update.go#L122-L141